scss/_variables.scss

```diff
@@ -454,17 +454,17 @@ $body-text-align:           null !default;
 //
 // Style anchor elements.
 
-$link-color:                              $black !default;
+$link-color:                              var(--link-color, $black) !default; // Boosted mod
```

Comment:
global remark for all css var, maybe need to prefix it by boosted-variable? : `var(--#{$boosted-variable-prefix}`